### PR TITLE
design-docs: timetable v2

### PR DESCRIPTION
**Pull request by multun**

### CODE REVIEW

File: content/docs/reference/design-docs/timetable.en.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+# Train schedule v2
+
+## Creation fields
+
+```yaml
+timetable_id: 42
+```
+
+## Modifyable fields
```
Comment (Khoyo) [inline content/docs/reference/design-docs/timetable.en.md:16]:
```suggestion
## Modifiable fields
```

**Review by flomonster — APPROVED** [review]

File: content/docs/reference/design-docs/timetable.en.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+POST /v2/train_schedule
+GET /v2/train_schedule/ID
+GET /v2/train_schedule/ID/path
+PATCH /v2/train_schedule/ID
+DELETE /v2/train_schedule/ID
+
+POST /v2/infra/ID/pathfinding/topo
+POST /v2/infra/ID/pathfinding/blocks
+
+# all the endpoints in path_properties takes a path (the output of pathfinding/blocks),
```
Comment (clarani) [inline content/docs/reference/design-docs/timetable.en.md:82]:
```suggestion
# takes a path (the output of pathfinding/blocks),
```

**Comment by multun** [comment]
@flomonster @clarani @Khoyo thanks for the review :) I've submitted an amendment to this proposal, which changes how margins and arrival times are defined https://github.com/osrd-project/osrd-website/pull/164 before merging

**Comment by flomonster** [comment]
## Note meeting with PO (Alex + Giuliana)

- Are engineering margins useful (can be defined by scheduled points)?
  - If not, we can get rid of it.
- We need the standard margin system whatsoever. It compromises the suggestion here: https://github.com/osrd-project/osrd-website/pull/164

File: content/docs/reference/design-docs/timetable.en.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+- when deleting a waypoint, the end user may want to preserve the associated stop. Making the separation clear in the data model helps with implementing this behavior correctly, if deemed relevant
+- bundling stops into the path makes it harder to describe what fields `path` waypoints should have, and what should have a separate object and reference. It was decided that keeping `path` a simple list of `Location`, with no strings attached, made things a little clearer.
+
+
+### No more engineering margins?
+
+In the legacy model, we had engineering margins. These margins had the property of being able to overlap. It was also possible to choose the distribution algorithm for each margin individually.
+
+We asked users to comment on the difference and the usefulness of retaining these margins with scheduled points. The answer is that there is no fundamental difference, and that the additional flexibility offered by engineering margins makes no pratical sense (overlap and choice of distribution...).
```
Comment (flomonster) [inline content/docs/reference/design-docs/timetable.en.md:52]: Wait for @bgiuliana confirmation.

Comment (flomonster) [reply content/docs/reference/design-docs/timetable.en.md:52]: @bgiuliana Verified this information.

File: content/docs/reference/design-docs/timetable/index.en.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+- The train path changed
+- The train running time changed
+
+What we can do about outdated trains:
+
+1. Nothing, they're updated without notification
+2. We can notify the user that a train schedule is outdated:
+    - Nothing can be done except acknoledge the change
+    - We can not check what changed between the old and new version
+    - We can not know the cause of this change (RS, Infra, Algorithms...)
```
Comment (flomonster) [inline content/docs/reference/design-docs/timetable/index.en.md:99]: Ask @bgiuliana about it.

**Review by flomonster — APPROVED** [review]
Great!

**Review by flomonster — APPROVED** [review]

**Pull request merged**